x-pack/plugins/security_solution/public/graphql/types.ts

```diff
@@ -96,9 +96,9 @@ export interface TlsSortField {
 }
 
 export interface PageInfoTimeline {
-  pageIndex: number;
+  pageIndex?: Maybe<number>;
```

Comment:
I am curious to understand why we had to change that

Reply:
as I want to be able to get all the timelines here x-pack/plugins/security_solution/server/lib/detection_engine/scripts/timelines/get_all_timelines.sh

Reply:
Angela and I discussed it, we will take care of it in the following PR by querying simply twice to get the timeline count to get all the timeline. Please create an issue to track it by labeling it `7.9

Reply:
https://github.com/elastic/kibana/issues/69972